test/extended/authorization/rbac/groups_default_rules.go

```diff
@@ -53,6 +53,7 @@ const (
 	oauthGroup    = oauth.GroupName
 	projectGroup  = project.GroupName
 	templateGroup = template.GroupName
+	csvGroup      = "operators.coreos.com"
```

Comment:
Note: this is an upstream OLM API, and doesn't live in o/api. It's probably best to hardcode here vs. introducing a new dependency into origin.

Reply:
name it operatorsCoreOSGroup, please

Reply:
Can do!

Reply:
Done